bin/nest-server-mpi

```diff
@@ -25,6 +25,8 @@ import sys
 import nest
 import nest.server
 
+HOST="${NEST_SERVER_HOST:-127.0.0.1}"
+PORT="${NEST_SERVER_PORT:-52425}"
```

Comment:
Shouldn't this be using `os.getenv()`? This looks a lot like bash syntax to me, while this is a Python script... Or am I missing something here?

Reply:
@joug You are right. I apologize for this mistake. 

Indeed such codes is wrong in Python script.
I will fix it.